fuel-vm/src/interpreter/executors/instruction.rs

```diff
@@ -641,7 +641,7 @@ where
                 let (a, b, c, d) = call.unpack();
 
                 // Enter call context
-                self.prepare_call(r!(a), r!(b), r!(c), r!(d))?;
+                self.prepare_call(a, b, c, d)?;
```

Comment:
Yep, I moved it to https://github.com/FuelLabs/fuel-vm/pull/487. It is here because without it the corresponding PR doesn't compile on `fuel-core` side=D